### PR TITLE
Add horizontal list

### PR DESCRIPTION
**Pull request by terencehuynh**
Love your work. This is my first time committing to this repo, and decided to make add something small for the first thing I do.

Please like and subscribe.

### CODE REVIEW

File: src/List/List.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ Specify if you want your list items to have a style type with the `listStyleType
 
 ## Ordered lists
 
-To use an unordered list, use the `isOrdered` prop.
+To use an ordered list, use the `isOrdered` prop.
```
Comment (jxom) [inline src/List/List.mdx:57]: oops!

File: src/List/styled.js [hunk 2]
```diff
@@ -19,6 +19,17 @@ const orderedProperties = css`
   }
 `;
 
+const horizontalProperties = css`
+  & li {
+    display: inline-block;
+    margin-right: 1rem;
```
Comment (jxom) [inline src/List/styled.js:25]: Lets maybe unapply the default margin-bottom styling we get with vertical lists

```suggestion
    margin-right: 1rem;
    margin-bottom: unset;
```

**Review by jxom — CHANGES_REQUESTED** [review]
Nice! Just one comment

File: src/List/styled.js (hunk 2 above)
Comment (terencehuynh) [reply src/List/styled.js:25]: Done 👍

**Pull request merged**